build/amis/ansible/roles/containerd/files/etc/crictl.yaml

```diff
@@ -0,0 +1 @@
+runtime-endpoint: unix:///run/containerd/containerd.sock
```

Comment:
We should probably configure containerd to use systemd as the cgroup manager. I can do that as a followup.